# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/DBSchema.cpp
# Commit: 1700d5b79273
# Full Hash: 1700d5b79273a6f34d67b7d16470973f0cb65793
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-21 21:21:23
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 31888ffde37a -r 1700d5b79273 dom/cache/DBSchema.cpp
--- a/dom/cache/DBSchema.cpp	Tue Dec 20 20:56:15 2022 +0000
+++ b/dom/cache/DBSchema.cpp	Tue Dec 20 20:56:15 2022 +0000
@@ -342,7 +342,9 @@
         nsIContentPolicy::TYPE_INTERNAL_FRAME_MESSAGEMANAGER_SCRIPT == 53 &&
         nsIContentPolicy::TYPE_INTERNAL_FETCH_PRELOAD == 54 &&
         nsIContentPolicy::TYPE_UA_FONT == 55 &&
-        nsIContentPolicy::TYPE_WEB_IDENTITY == 57,
+        nsIContentPolicy::TYPE_WEB_IDENTITY == 57 &&
+        nsIContentPolicy::TYPE_INTERNAL_WORKER_STATIC_MODULE == 58 &&
+        nsIContentPolicy::TYPE_END == 59,
     "nsContentPolicyType values are as expected");
 
 namespace {